client/app/router.js

```diff
@@ -28,6 +28,7 @@ Router.map(function() { // eslint-disable-line
   this.route('show-project', { path: '/projects/:id' });
   this.route('show-geography', { path: '/projects' });
   this.route('disclaimer');
+  config.showAlerts && this.route('statuses');
```

Comment:
Can you fix this so we aren't breaking tests? You might just need to use an `if` instead of the `&&` shorthand.

Reply:
Sure thing